nocaselist/_nocaselist.py

```diff
@@ -489,3 +531,11 @@ def lower_key(value):
 
         super(NocaseList, self).sort(key=lower_key, reverse=reverse)
         self._lc_list = _lc_list(self)
+
+
+# Remove methods that should only be present in a particular Python version
+# If the documentation is built, the methods are not removed in order to show
```

Comment:
Think this should say if the documentation is being built.

Reply:
DONE